CONTRIBUTING.md

```diff
@@ -162,7 +162,7 @@ Then, check out a new branch with `git checkout -b my-branch-name`
 
 Create your code / documentation changes in this branch, and commit when done. Once done, `git push fork your-branch-name`
 
-Once this is finished pushing, you can go to your fork on GitHub. It should now ask you right away if you want to create a pull request. Clicking that button should set you up with a text field just like when creating a new issue on GitHub. Fill it out and submit the pull request. Then, we can review it “Pull requests” in this original repository.
```

Comment:
I know this isn't really a problem word in this case, but felt it was better to be able to leave the strong check than add an ignore to the file or sentence.